### ISSUE TITLE
Translation to Norwegian (GUI)

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-gui#289]
For some messages in the GUI itself translation to Norwegian is incomplete. The following items in [src/assets/i18n/nb.json] needs translation. Translation can either be provided as a pull request or as a comment to this issue. This issue replaces (incorporates) issue #246.

In [src/assets/i18n/nb.json]:
```json
    "Algorithm required": null,
    "All": null,
    "DELEGATED_PLURAL": null,
    "Digest type required": null,
    "Disable IPv4": null,
    "Disable IPv6": null,
    "NS required": null,
    "TOOLTIP_ALL": null,
    "TOOLTIP_CRITICAL": null,
    "TOOLTIP_ERROR": null,
    "TOOLTIP_INFO": null,
    "TOOLTIP_NOTICE": null,
    "TOOLTIP_WARNING": null,
    "UNDELEGATED_PLURAL": null,
    "UNDELEGATED_SINGULAR": null,
```

When the key is a TAG then the translation should be a translation of the value in [src/assets/i18n/en.json]:
```json
    "DELEGATED_PLURAL": "Delegated",
    "TOOLTIP_ALL": "Display all results",
    "TOOLTIP_CRITICAL": "Critical - means a very serious error.",
    "TOOLTIP_ERROR": "Error - means a problem that is very likely (or possibly certain) to negatively affect the function of the zone being tested, but not so severe that the entire zone becomes unresolvable",
    "TOOLTIP_INFO": "Info - is something that may be of interest to the zone's administrator but that definitely does not indicate a problem.",
    "TOOLTIP_NOTICE": "Notice - means something that should be known by the zone's administrator but that need not necessarily be a problem at all",
    "TOOLTIP_WARNING": "Warning - means something that will under some circumstances be a problem, but that is unlikely to be noticed by a casual user.",
    "UNDELEGATED_PLURAL": "Undelegated",
    "UNDELEGATED_SINGULAR": "Undelegated",
```

[src/assets/i18n/nb.json]: https://github.com/zonemaster/zonemaster-gui/blob/develop/src/assets/i18n/nb.json
[src/assets/i18n/en.json]: https://github.com/zonemaster/zonemaster-gui/blob/develop/src/assets/i18n/en.json

**Comment by matsduf** [comment]
Resolved by #246

**Comment by matsduf** [comment]
@repperille, can you look at this?

**Comment by matsduf** [comment]
Resolved by #294
